From 9637c35339111d92add461fc3ad2a032e90abc9c Mon Sep 17 00:00:00 2001
From: psykose <alice@ayaya.dev>
Date: Fri, 4 Oct 2024 18:18:29 +0200
Subject: [PATCH] meson: fix source version outside git

when git is not found the whole conf var is skipped, and that fails to compile
---
 meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 08fb1bc141c..20241cef418 100644
--- a/meson.build
+++ b/meson.build
@@ -18,6 +18,7 @@ conf.set('MICRO_VERSION', fwupd_micro_version)
 conf.set_quoted('PACKAGE_VERSION', fwupd_version)
 
 # get source version, falling back to package version
+source_version = fwupd_version
 git = find_program('git', required: false)
 tag = false
 if git.found()
@@ -26,8 +27,8 @@ if git.found()
     source_version = fwupd_version
   endif
   tag = run_command([git, 'describe', '--exact-match'], check: false).returncode() == 0
-  conf.set_quoted('SOURCE_VERSION', source_version)
 endif
+conf.set_quoted('SOURCE_VERSION', source_version)
 
 # libtool versioning - this applies to libfwupd
 #
